docker image taste path mismatch

**Describe the bug**
When building a dev docker image, the docker container will create /etc/strelka/taste.yara instead of the /etc/strelka/**taste/**taste.yara path that is referenced in the strelka.yml file (and matches the project repo folder structure).  This results in a "No such file or directory" error when submitting new files to the server for processing.

**Steps to reproduce**
1) git clone https://github.com/target/strelka /opt/strelka
2) cd /opt/strelka && docker build -t strelka-dev .
3) docker run --rm -v /tmp/sample_files:/tmp/sample_files strelka-dev strelka.py
4) run 'docker stats' to get $containerid
5) docker exec $containerid strelka_user_client.py --broker 127.0.0.1:5558 --path /tmp/sample_files/file0001
Error message will be present as an [ERROR] event on the screen where the container launched in step 3 is running.

**Expected behavior**
Files should be tasted and assigned to scanners properly.  When bug is present, tasting fails, no worker is assigned by the broker, no scanning occurs.

**Screenshots**
N/A

**Server and project version**
OS: Xubuntu 18.04
Commit hash: N/A (current master as of Feb 25, 2019)

**Additional context**
N/A

--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,7 @@ setup(
     description="strelka: scanning files at scale with python and zeromq",
     license="Apache 2.0",
     packages=find_packages(),
-    data_files=[("/etc/strelka", ["etc/strelka/strelka.yml", "etc/strelka/pylogging.ini", "etc/strelka/taste/taste.yara", "etc/dirstream/dirstream.yml"])],
+    data_files=[("/etc/strelka", ["etc/strelka/strelka.yml", "etc/strelka/pylogging.ini", "etc/dirstream/dirstream.yml"]), ('/etc/strelka/taste/', ['etc/strelka/taste/taste.yara'])],
     scripts=["strelka.py", "strelka_dirstream.py", "strelka_user_client.py", "generate_curve_certificates.py", "validate_yara.py"],
     zip_safe=False
 )

